coalib/bearlib/__init__.py

```diff
@@ -5,14 +5,23 @@
 """
 
 import logging
-from functools import wraps
 
 from coalib.settings.FunctionMetadata import FunctionMetadata
 
 
 def _do_nothing(x): return x
 
 
+def profile_bears_decorator(func):
```

Comment:
@ishanSrt  need your help here, I am failing in understanding this change.

Reply:
I am enabling the profiler hear so that it doesn't contain the overhead of the function calls from the decorator itself and then actually calling the run method using this additional decorator

Reply:
so if we want to profile our bear it will add this extra decorator which adds `prof.enable()` just before calling the `run()` method 😄

Reply:
you will need to do the same for the decorators for linters

Reply:
Got it thanks for the help :+1: